hoot-core/src/main/cpp/hoot/core/algorithms/DirectionFinder.h

```diff
@@ -32,10 +32,6 @@
 
 #include <hoot/core/OsmMap.h>
 
-namespace hoot {
-  class Way;
-}
-
```

Comment:
Quite a few of the changes were removing unnecessary forward declarations.  That is what got me started on this clean up in the first place.  They Qube said to remove duplicate forward declarations.